workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/services/actions/PublisherAsset.ts

```diff
@@ -45,7 +48,8 @@ export default class PublisherAsset {
 		if (!appFolderId) {
 			const packageFolder = await HeadlessDelivery.createDocumentFolder(
 				folderName,
-				publisherFolderId
+				publisherFolderId,
+				'Members'
```

Comment:
Create a variable for this or a enum with the folder permissions

So we know that "Members" is actually a role definition